developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md

```diff
@@ -46,8 +46,10 @@ services:
       HUGGINFACE_APIKEY: sk-foobar # request a key on huggingface.co, setting this parameter is optional, you can also provide the API key on runtime
       CLUSTER_HOSTNAME: 'node1'
 ```
-
-* Note: you can also use the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+:::note
+* You can also use the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+* Starting with `v1.11.0` the `HUGGINFACE_APIKEY` variable is now optional and you can instead provide the key at insert/query time as an HTTP header.
```

Comment:
* Should we mention the name of the HTTP header?
* Should we consistently format version numbers as code? The vast majority I've seen were plain text.